### PR TITLE
Median and percentile

### PR DESCRIPTION
**Pull request by zemogle**
This was requested by Tim Lister. The current thumbnails don't have enough contrast to see faint asteroids.

### CODE REVIEW

File: thumbservice.py [hunk 1]
```diff
@@ -69,14 +69,8 @@ def frames_for_requestnum(reqnum, request):
     frames = requests.get(
         '{0}frames/?REQNUM={1}'.format(ARCHIVE_API, reqnum),
         headers=headers
-    ).json()['results']
```
Comment (Fingel) [inline thumbservice.py]: Why was this removed? This is for color images, so that only frames with the same reduction level will be combined.

**Review by Fingel — CHANGES_REQUESTED** [review]

File: thumbservice.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 from flask import Flask, request, abort, jsonify, send_file
-from flask_cors import CORS
+from flask.ext.cors import CORS
```
Comment (Fingel) [inline thumbservice.py]: This results in deprecation warnings. Make sure your flask installation is up to date.

**Comment by zemogle** [comment]
I think I've fixed the points you've raised. I think this happened because a botched merge. Sorry about that.

**Review by Fingel — APPROVED** [review]
Looks good
